JustSaying/AwsTools/MessageHandling/SqsQueueBase.cs

```diff
@@ -12,7 +13,7 @@ namespace JustSaying.AwsTools.MessageHandling
     public abstract class SqsQueueBase
     {
         public string Arn { get; protected set; }
-        public string Url { get; protected set; }
+        public Uri Url { get; protected set; }
```

Comment:
`QueueUri`? I'd suggest `Uri`, but then it's the same name as the type.

Reply:
I'm in favour of calling it `Uri` since often it is used as e.g. ` _queue.Uri`. 
Second choice would be `QueueUri` to match the name in message.